### DISCUSSION TITLE
Linkace behind reverse proxy/different subdir or different port

### DISCUSSION THREAD

**Discussion by janheise** [Kovah/LinkAce#679] [Support]
Hi, 

I'm trying to install the simple docker setup behind an nginx reverse proxy. I tried two setups:
1. under a different URL like https://server.name/linkace instead of "/" -> always drops the "linkace" from the URL when opening a page
2. with a different port like https://server.name:444 -> always drops the port and reverts to the regular HTTPS port

so, both ways are not working, setting APP_URL in .env also does not seem to change anything. Did someone accomplish sth. like that?

**Comment by Kovah** [comment]
Hi, it partly depends on your nginx proxy configuration. Running LinkAce in a sub-directory like `server.name/linkace` is currently not supported, but with a port should work.

Something like this should work.
```
server {
    listen 444;
    server_name site.name;
    port_in_redirect off;

    location / {
        proxy_pass http://127.0.0.1:8080; # LinkAce in Docker with port 
        proxy_set_header X-Forwarded-For $remote_addr;
        proxy_set_header X-Forwarded-Proto $scheme;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header Host $host;
    }
}
```
However, I recommend using a subdomain. It's the easiest solution and works without sub-directories or ports.

**Reply by Mek101** [reply]
Would it be possible to add support for a sub-directory?

**Comment by janheise** [comment]
Thanks for chiming in!

I am not able to use a subdomain. 

Your config snippet looks quite similar to mine, it's not working - at least here it's not: the port get's cut when when calling `/` and the redirect to `/login`

**Reply by Kovah** [reply]
Could you try to replace
```
proxy_set_header Host $host;
```
with
```
proxy_set_header Host $host:$server_port;
```
Not sure if that really helps, have not tested this.

**Reply by Mek101** [reply]
Does not work
